tests/nn/models/gate_points_2101_test.py

```diff
@@ -27,13 +36,72 @@ def test_gate_points_2101(float_tolerance):
         num_nodes=num_nodes,
     )
 
-    # Test equivariance:
+    def random_graph():
+        N = random.randint(3, 7)
+        return {
+            'pos': torch.randn(N, 3),
+            'x': f.irreps_in.randn(N, -1),
+            'z': f.irreps_node_attr.randn(N, -1)
+        }
+
+    return f, random_graph
+
+
+def test_convolution_jit(network):
+    f, _ = network
+    # Get a convolution from the network
+    assert_auto_jitable(f.layers[0].first)
+
+
+def test_gate_points_2101_equivariant(network):
+    f, random_graph = network
+    # -- Test equivariance: --
     def wrapper(pos, x, z):
         data = Data(pos=pos, x=x, z=z, batch=torch.zeros(pos.shape[0], dtype=torch.long))
         return f(data)
 
     assert_equivariant(
         wrapper,
-        irreps_in=['cartesian_points', irreps_in, irreps_attr],
-        irreps_out=[irreps_out],
+        irreps_in=['cartesian_points', f.irreps_in, f.irreps_node_attr],
+        irreps_out=[f.irreps_out],
     )
+
+
+def test_gate_points_2101_jit(network):
+    f, random_graph = network
+
+    with warnings.catch_warnings():
+        warnings.filterwarnings('error', category=torch.jit.TracerWarning)
+        f_traced = trace(
+            f,
+            example_inputs=(random_graph(),),
+            # Check the compute graph on 3 other random inputs
+            check_inputs=[(random_graph(),) for _ in range(3)]
+        )
+
+    dat = random_graph()
+    assert torch.allclose(f(dat), f_traced(dat))
+
+    # - Try saving, loading in another process, and running -
+    with tempfile.TemporaryDirectory() as tmpdir:
+        # Save stuff
+        f_traced.save(tmpdir + "/model.pt")
+        torch.save(dat, tmpdir + '/dat.pt')
+        # Load in new process
+        with open(tmpdir + '/code.py', 'x') as code:
+            code.write(f"""
+import torch
+# Needed for the TorchScript kernels for scatter and radius_graph
+import torch_scatter
+import torch_cluster
+f = torch.jit.load('{tmpdir}/model.pt')
+d = torch.load('{tmpdir}/dat.pt')
+out = f(d)
+torch.save(out, '{tmpdir}/out.pt')
+""")
+        # Run
+        proc_res = subprocess.run(['python', tmpdir + '/code.py'])
```

Comment:
Maybe there is a way to obtain the currently used python interpreter

Reply:
Resolved in e22a35b